### ISSUE TITLE
bulk invite GitHub watchers and stargazers

### ISSUE THREAD

**Issue by jab** [gitterHQ/gitter#1636]
I have an open source project with several watchers and stargazers. Many of them have been interested to join the Gitter chat for my project, but the Gitter badge in my README wasn't a sufficient way to get them to join; it took manually inviting them from Gitter's "add people to this room" UI. Would it be worth adding a button to Gitter to bulk invite all the watchers and stargazers of a project, so interested followers could be invited easily, rather than one-by-one?

Thanks for your consideration, and thanks for Gitter!

**Comment by mydigitalself** [comment]
Hi there,

Not a bad suggestion, but a few concerns/considerations:

1) If the user is not on Gitter, unless they have a public email on GitHub, we don't have any way to invite them. You'll find the majority of people don't really have public addresses.

2) Invite spam can also be a problem and we need to be considerate of making it too easy to bulk email a large number of people.

**Comment by jab** [comment]
I had those concerns too. For whatever it's worth, in my experience with my project, most of the watchers and stargazers I've manually invited have (1) had their emails public, and (2) accepted the invitation. And not a single person has ever complained when I invited them. Maybe starring/watching is enough of an expression of interest in the project that an invitation to join the chat doesn't feel spammy. (And watching seems an even stronger signal than starring, since it signifies interest in receiving email updates anytime someone opens an issue, PR, comments, etc.)

I also use Gitter in lieu of e.g. a users mailing list, etc. So if a user wants to follow or participate in any discussions that I start to ask users for feedback about possible changes for an upcoming version, Gitter is the only way to do it.

Just one data point, but for whatever it's worth!
